tests/cli/test_init_sqlite.py

```diff
@@ -616,11 +626,7 @@ def test_init_on_existing_project_with_datasource_with_no_suite_create_one(
         result = runner.invoke(
             cli,
             ["init"],
-            input="\n1\n{schema}\n{table}\nsink_me\n\n\n\n".format(
-                os.path.join(project_dir, "data/Titanic.csv"),
```

Comment:
Was this manually changed or changed by ruff?

Reply:
I don't remember doing this.
But it looks like a correct change.
The format string wasn't using the `os.path.join(project_dir, "data/Titanic.csv")` param at all.